bin/diary

```diff
@@ -1,7 +1,58 @@
 #!/user/bin/env python
 
+import sys, os, sqlite3
+
+# Main function - passes commands into their own functions
 def diary():
-    print('Coming soon...')
+    if '-h' in sys.argv or '--help' in sys.argv:
+        help()
+    elif 'generate' in sys.argv:
+        generate(sys.argv)
+    else:
+        print("Diary couldn't parse that command.")
+        help()
+
+# Help command
+def help():
```

Comment:
This shadows a builtin name, could you name this diary_help or something along those lines.